frontend/app/controllers/Joiner.scala

```diff
@@ -72,8 +73,13 @@ object Joiner extends Controller with ActivityTracking
       customSignInUrl=Some(signInUrl)
     )
 
-    Ok(views.html.joiner.tierChooser(TouchpointBackend.Normal.catalog, pageInfo, eventOpt, accessOpt, signInUrl))
-      .withSession(request.session.copy(data = request.session.data ++ contentRefererOpt.map(JoinReferrer -> _)))
+    contentRefererOpt.filter(Uri.parse(_).host.exists(_ == "www.theguardian.com")).map { contentURL =>
```

Comment:
Might be OK for now... but I could see that really coming back to bite someone in the future!